BUILD.md

```diff
@@ -4,9 +4,12 @@ Build and Package vpaw
 This document summarizes how to build and package vpaw on Windows. Instructions for Linux and macOS are similar.
 For more details, see [3D Slicer Developer Wiki](http://wiki.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers)
 
-Prerequisites 
--------------
+vpaw is a custom Slicer application. Reading the [3D Slicer Developer Documentation](https://slicer.readthedocs.io/en/latest/developer_guide/index.html) may help answer additional questions.
```

Comment:
Consider updating this document based of https://github.com/KitwareMedical/SlicerCustomAppTemplate/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/BUILD.md

Reply:
It is generated from the file you mention, but I have retained a few bullet points from the previous version, do you want me to remove them? bullet points listed below:
```
* Microsoft Windows 7 or above recommended

* Supported Microsoft Visual Studio versions:
    * Visual Studio 2015
    * Visual Studio 2017

* [CMake](http://cmake.org/cmake/resources/software.html), version 3.11 or above

* Qt, version 5.10 or above

* [Git](http://git-scm.com/downloads)

```